content/post/115-protocol-labs-and-igalia-collaborating-on-dweb-in-browsers.md

```diff
@@ -0,0 +1,194 @@
+---
+date: 2021-01-15
+title: Protocol Labs and Igalia collaborating on dweb in browsers
```

Comment:
Since this is on the IPFS blog and not the Protocol Labs’ one, should we say “IPFS and Igalia collaborating...” ?